erpnext/crm/doctype/lead/lead.py

```diff
@@ -390,3 +391,26 @@ def set_missing_values(source, target):
 		}
 	}, target_doc, set_missing_values)
 	return target_doc
+
+def rearrange_standard_fields():
+	"""Rearrange standard field in lead doctype"""
+	count = 1
+	for df in frappe.get_meta("Lead").get("fields"):
+		if df.fieldname in ["territory", "address_html", "contact_html", "contact_by"]:
+			count= count + 1
+			frappe.db.sql("""UPDATE `tabDocField` SET idx={0} WHERE fieldname=%s and parent='Lead' """.format(count), df.fieldname, as_dict=True)
```

Comment:
Instead of re-arranging the standard field everytime after migrate, change it directly in lead.json of erpnext app once and delete this code.